monitor/xfeemngr/tokenprice/buffer.go

```diff
@@ -68,6 +86,8 @@ func (b *Buffer) stream(ctx context.Context) {
 			}
 
 			b.setPrice(token, price)
+
+			log.Info(ctx, "Updating token price", "token", token, "old", buffed, "new", price)
```

Comment:
nit: I'd change this to a metric and a debug log.

Reply:
Agreed this will be a metric - and then I'll remove logs. Just wanted some visibility in it for now